src/pirate/psu.c

```diff
@@ -98,18 +98,24 @@ void psu_set_i(float current, struct psu_status_t* psu) {
 }
 
 void psu_dac_set(uint16_t v_dac, uint16_t i_dac) {
+#if (BP_VER != 7)

```

Comment:
As above, at minimum, add comment with `//BUGBUG - TODO - ` so indicate the why ... and so it can then be changed to capability-based.